packages/web-components/src/components/va-icon/va-icon-global.ts

```diff
@@ -0,0 +1,19 @@
+declare global {
```

Comment:
define a global init function along with a getter and setter to set the icon path.